lib/lotus/action/cookie_jar.rb

```diff
@@ -131,6 +137,7 @@ def extract(env)
         hash   = env[COOKIE_HASH_KEY] ||= {}
         string = env[HTTP_HEADER]
 
+        return if string && string.include?(RACK_SESSION_KEY)
```

Comment:
yes, you are right. the `Set-Cookie` header value for rack session is `rack.session=hash`. maybe string isn't the correct var to check, maybe it's hash, what do you think @pascalbetz ?